packages/api/scripts/run-test.sh

```diff
@@ -9,10 +9,6 @@ $CLI db-sql --cargo --testing --reset
 echo "creating minio bucket..."
 $CLI minio bucket create dotstorage-dev-0
 
-cd $THIS_DIR/..
```

Comment:
Moved out `tsc` checking to a `posttest` run script to separate it from our tests, but still have it run after the tests have run by the magic of npm lifecycle hook scripts, so I can run the tests without appeasing tsc first.